ecida-operator/Makefile

```diff
@@ -0,0 +1,142 @@
+# VERSION defines the project version for the bundle. 
```

Comment:
this is the first time I review an 'actual' make file. THere's stuff in here I've never seen before, but nice. And iirc it was auto generated right?